### PR TITLE
Signal message format change for simulcast streams

### PR DESCRIPTION
**Pull request by starwarfan**
Signal message format change for simulcast streams

### CODE REVIEW

File: doc/Client-Portal Protocol.md [hunk 1]
```diff
@@ -212,7 +212,11 @@ This a format for client reconnects.
                     }
                     | undefined
                  }
-                 | undefined
+                 | undefined,
+               alternative: [{ /* for simulcast streams */
```
Comment (jianjunz) [inline doc/Client-Portal Protocol.md]: Will `alternative.parameters` includes `parameters`?

Comment (starwarfan) [reply doc/Client-Portal Protocol.md]: I mean,
{ format: f1, parameters: { resolution: r1 }， // this is default one in simulcast
  alternative: [
    { format: f2, parameters: { resolution: r2 },  // this is another one can be chose in simulcast
  ]
}
If user subscribes this stream without any parameters, the default one will be used

Comment (jianjunz) [reply doc/Client-Portal Protocol.md]: OK. FYI, Client SDK may have a different data structure to represent simulcast streams. All simulcast formats will be included in `PublicationSettings`.

Comment (xiandeatintel) [reply doc/Client-Portal Protocol.md]: How about change object(VideoInfo) into:
object(VideoInfo)::                                                  
              {                                                                  
               status: "active" | "inactive" | undefined,                        
               source: "camera" | "screen-cast" | "raw-file" | "encoded-file" | undefined,
               original: [{                                                      
                 format: object(VideoFormat),                                    
                 parameters: object(VideoParameters) | undefined,                
                 simulcastRid: string(SimulcastRid) | undefined                  
               },                                                                
               optional:                                                         
                 {                                                               
                  format: [object(VideoFormat)] | undefined,                     
                  parameters:                                                    
                    {                                                            
                     resolution: [object(Resolution)] | undefined,               
                     framerate: [number(FramerateFPS)] | undefined,               
                     bitrate: [number(BitrateKbps)] | [string(BitrateMultiple)] | undefined,
                     keyFrameInterval: [number(KeyFrameIntervalSecond)] | undefined
                    }                                                            
                    | undefined                                                  
                 }                                                               
                 | undefined                                                     
              }

Comment (starwarfan) [reply doc/Client-Portal Protocol.md]: Would non-simulcast streams stay unchanged?

Comment (starwarfan) [reply doc/Client-Portal Protocol.md]: @jianjunz what structure of stream will be like in client sdk?

Comment (xiandeatintel) [reply doc/Client-Portal Protocol.md]: > Would non-simulcast streams stay unchanged?

No, non-simulcast streams have the same description, but the list of 'original' has only one item with 'simulcastRid' being set to undefined.

**Pull request merged**